packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/ContentEdit.ts

```diff
@@ -55,17 +56,25 @@ export default class ContentEdit implements EditorPlugin {
                 (hasSettingForKey && this.settingsOverride[key]) ||
                 (!hasSettingForKey && !feature.defaultDisabled)
             ) {
-                features.push(feature);
+                this.features.push(feature);
             }
         });
-
-        features
+        this.features
             .concat(this.additionalFeatures || [])
```

Comment:
Need to save the result of concat() to the plugin, so we can remove it when dispose as well